sdk/storage/Azure.Storage.Files.Shares/tests/ShareClientTests.cs

```diff
@@ -440,6 +441,30 @@ public async Task GetStatisticsAsync()
             Assert.IsNotNull(response);
         }
 
+        [Test]
+        public async Task GetStatisticsAsync_LargeShare()
+        {
+            // Arrange
+            long size = 3 * (long)Constants.GB;
+            MockResponse mockResponse = new MockResponse(200);
+            mockResponse.SetContent($"﻿<?xml version=\"1.0\" encoding=\"utf-8\"?><ShareStats><ShareUsageBytes>{size}</ShareUsageBytes></ShareStats>");
+            ShareClientOptions shareClientOption = new ShareClientOptions()
+            {
+                Transport = new MockTransport(mockResponse)
+            };
+            ShareClient shareClient = new ShareClient(new Uri(TestConfigDefault.FileServiceEndpoint), shareClientOption);
```

Comment:
This still needs to be instrumented for the sync/async to work correctly.

Reply:
fixed.